docker_build.sh

```diff
@@ -16,8 +16,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -e
 echo $@
 
+if [[ ! -x `which jq` ]]; then echo "Please install \"jq\" to continue"; exit 1; fi
```

Comment:
Looks like this is a dup of the line below?

Reply:
thank you. comment addressed